spec/factories/link_change.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :link_change do
+    source_content_id { SecureRandom.uuid }
+    target_content_id { SecureRandom.uuid }
+    action
+    link_type "taxons"
+    change 1
```

Comment:
I think this should be your enum rather than an int

Reply:
Amended.